examples/project/classic-project.yaml

```diff
@@ -9,6 +9,6 @@ spec:
   projectTemplateKey: com.atlassian.servicedesk:itil-v2-service-desk-project
   description: "Sample project for jira-service-desk-operator"
   assigneeType: PROJECT_LEAD
-  leadAccountId: 5ebfbc3ead226b0ba46c3590
+  leadAccountId: 5f62e5902b42470070d1fb83
```

Comment:
You are not supposed to update lead account id here since this is for demonstration purposes for end users.